$NetBSD: patch-glslc_CMakeLists.txt,v 1.1 2026/08/27 11:17:19 adam Exp $

Get package version from pkgsrc, to avoid scanning third_party dir.

--- glslc/CMakeLists.txt.orig	2026-08-26 18:38:57.273819375 +0000
+++ glslc/CMakeLists.txt
@@ -54,7 +54,6 @@ if(SHADERC_ENABLE_EXECUTABLES)
   target_include_directories(glslc_exe PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/.. ${spirv-tools_SOURCE_DIR}/include)
   set_target_properties(glslc_exe PROPERTIES OUTPUT_NAME glslc)
   target_link_libraries(glslc_exe PRIVATE glslc shaderc_util shaderc)
-  add_dependencies(glslc_exe build-version)
 endif(SHADERC_ENABLE_EXECUTABLES)
 
 shaderc_add_tests(
